src/Modal.js

```diff
@@ -30,13 +92,20 @@ const Modal = React.createClass({
       backdrop: true,
       keyboard: true,
       animation: true,
-      closeButton: true
+      closeButton: true,
+      enforceFocus: true
     };
   },
 
+  getInitialState(){
+    return { };
```

Comment:
It is pretty much the same yeah, except that providing an empty object lets you access properties on it for a falsey value instead of an error (Null Exception)
```js
// without the empty object in getInitialState this would throw the first render
var style = this.state.dialogStyles // state is null
```

I could be slightly more correct and set state to `{ dialogStyles: undefined }` but it seems unneeded